tests/e2e/specs/onboarding/step-3-complete-campaign.test.js

```diff
@@ -495,6 +495,64 @@ test.describe( 'Complete your campaign', () => {
 		} );
 	} );
 
+	test.describe( 'Enhanced conversion prompt should appear as last step in setup success modal', () => {
+		test.beforeAll( async () => {
+			await page.evaluate( () => window.sessionStorage.clear() );
+			await setupAdsAccountPage.mockAdsAccountConnected();
+			await completeCampaign.mockCompleteAdsSetup();
+			await completeCampaign.goto();
+			await completeCampaign.clickSkipPaidAdsCreationButton();
+			await completeCampaign.clickCompleteSetupModalButton();
+		} );
+
+		test( 'should see the setup success modal', async () => {
+			const setupSuccessModal = completeCampaign.getSetupSuccessModal();
+			await expect( setupSuccessModal ).toBeVisible();
+		} );
+
+		test( 'should have three prompts in the setup success modal', async () => {
```

Comment:
@ankitrox This is a bit confusing. In the parent test description, it tests whether EC is the last step, which technically means there should be only 2 screens in the setup success modal (the last screen is not rendered if ads setup is complete). Can you check again please?

Reply:
I have fixed this and added both scenarios where ads setup is completed and incomple.